.travis.yml

```diff
@@ -3,4 +3,4 @@ language: ruby
 sudo: false
 
 rvm:
-  - 1.9.3
+  - 2.2.4
```

Comment:
Current `travis.yml` file in `master` runs the build against several Ruby versions now :smile: